### PR TITLE
Add in output samplerate

### PR DESCRIPTION
**Pull request by armandobelardo**
Addresses #26

### CODE REVIEW

**Comment by armandobelardo** [comment]
@chrisstaite I was wondering if you've seen this issue before, I was trying to build locally to test out the change but I'm running into some cpp build issues

```
libtool: link: sed 's|^|_|' < ../include/libmp3lame.sym > .libs/libmp3lame-symbols.expsym
libtool: link: gcc -dynamiclib  -o .libs/libmp3lame.0.dylib  .libs/VbrTag.o .libs/bitstream.o .libs/encoder.o .libs/fft.o .libs/gain_analysis.o .libs/id3tag.o .libs/lame.o .libs/newmdct.o .libs/presets.o .libs/psymodel.o .libs/quantize.o .libs/quantize_pvt.o .libs/reservoir.o .libs/set_get.o .libs/tables.o .libs/takehiro.o .libs/util.o .libs/vbrquantize.o .libs/version.o .libs/mpglib_interface.o   -Wl,-force_load,../mpglib/.libs/libmpgdecoder.a  -lm  -O3 -arch x86_64 -mmacosx-version-min=10.6 -arch x86_64 -mmacosx-version-min=10.6   -install_name  /Users/armandobelardo/Git/oss/forks/lameenc/build/lame/src/lame/lib/libmp3lame.0.dylib -compatibility_version 1 -current_version 1.0 -Wl,-exported_symbols_list,.libs/libmp3lame-symbols.expsym
ld: Undefined symbols:
  _init_xrpow_core_sse, referenced from:
      _init_xrpow_core_init in quantize.o
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[6]: *** [libmp3lame.la] Error 1
make[5]: *** [all-recursive] Error 1
make[4]: *** [all-recursive] Error 1
make[3]: *** [all] Error 2
make[2]: *** [lame/src/lame-stamp/lame-build] Error 2
make[1]: *** [CMakeFiles/lame.dir/all] Error 2
make: *** [all] Error 2
```

File: setup.py [hunk 1]
```diff
@@ -66,10 +66,10 @@
     configuration['setup_requires'] = ['setuptools-git-versioning']
     configuration['setuptools_git_versioning'] = {
         'enabled': True,
-        'starting_version': '1.6.3'
+        'starting_version': '1.6.4'
```
Comment (chrisstaite) [inline setup.py:69]: Since it's a new feature, could you bump to 1.7.0 please.

File: lameenc.c [hunk 2]
```diff
@@ -48,6 +48,7 @@ static PyObject* Encoder_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
         }
         lame_set_num_channels(self->lame, 2);
         lame_set_in_samplerate(self->lame, 44100);
+        lame_set_out_samplerate(self->lame, 0);  // 0 is the default in lame
```
Comment (chrisstaite) [inline lameenc.c:51]: If this is the default already, there's no need to set it.

**Review by chrisstaite — CHANGES_REQUESTED** [review]
Looks good, just a couple of touch ups please.

File: lameenc.c (hunk 2 above)
Comment (armandobelardo) [reply lameenc.c:51]: Ah nice, will remove!

**Review by chrisstaite — APPROVED** [review]

**Pull request merged**